stackit/internal/services/ske/cluster/resource.go

```diff
@@ -771,28 +789,54 @@ func toHibernationsPayload(ctx context.Context, m *Cluster) (*ske.Hibernation, e
 }
 
 func toExtensionsPayload(ctx context.Context, m *Cluster) (*ske.Extension, error) {
-	if m.Extensions == nil {
+	if m.Extensions.IsNull() || m.Extensions.IsUnknown() {
 		return nil, nil
 	}
-	ex := &ske.Extension{}
-	if m.Extensions.Argus != nil {
-		ex.Argus = &ske.Argus{
-			Enabled:         conversion.BoolValueToPointer(m.Extensions.Argus.Enabled),
-			ArgusInstanceId: conversion.StringValueToPointer(m.Extensions.Argus.ArgusInstanceId),
-		}
+	ex := extensions{}
+	diags := m.Extensions.As(ctx, &ex, basetypes.ObjectAsOptions{})
+	if diags.HasError() {
+		return nil, fmt.Errorf("error in extensions object conversion %v", diags.Errors())
```

Comment:
```suggestion
		return nil, fmt.Errorf("converting extensions object: %v", diags.Errors())
```

The outtermost error message should already indicate that there is an error (see https://revolution.dev/app/-JqFGExX46gs9mH7vxR5/WORKSPACE_DOCUMENT/-Mvdp0Q7_oO2mmJTLpNI/)